### ISSUE TITLE
What is Recipr_ID ?

### ISSUE THREAD

**Issue by streckerja** [avollkopf/iSpindel-TCP-Server#13]
U
sung the TCP server, there is not a way to input recipe name into the database.  Field Recipe_ID in the Data table is an Integer.  The iSpindle doesn't provide this information and the Raspi TCP server doesn't have a way to input it.  Now, this being an integer, i would have expected a db file which associates the number to a name.  What were your expectations for use of this db field?
also, note that the installation of git-lib doesn't work with Buster..  it doesn't need it to clone the repository.

**Comment by avollkopf** [comment]
Have you tried the image for the Raspi? This is working. (Download link is in the readme)

Unfortunately, you have not provided plenty of information.

Are you using the lastest version of the script
Did you already send data from a spindel to your server?
Do you see the data on the index page?
Where and how do you want to enter the recipe name? 

Recipe_ID is an internal parameter that is created by the scripts automatically. It has to be an integer and is not the recipe name.

**Comment by streckerja** [comment]
Yes, I have the latest version and all is working.
I hadn't really thought about how I would implement it as I had assumed you
had something in the planning.  I realize that this is not a trivial task.

**Comment by avollkopf** [comment]
ok, it was probably too early this morning, when I read your message....

The recipe_id is in the data table and in the archive table. 

when you start a fermentation, a new row is added to the archive table and the recipe_id in this table is autoincrease by 1.
New data for from this spindle will be added with this particular recipe_id until you start a new fermentation.

Tihs allows to sort the data for the archive functions. I hope this helps a bit more than my first response.

**Comment by streckerja** [comment]
I was probably a bit premature with my question. I found where to enter the recipe name under the start fermentation menu. 
I am still calibrating my iSpindle so I haven’t had the chance to use it myself. I built a number of them for a brewing club in Nebraska. So I was anticipating questions I would get. I want to thank you for this version of the TCP software for the Raspberry Pi.  Now to get it running as a web site. 

Thank again

Jim Strecker
